internal/replacements/replacements_test.go

```diff
@@ -46,13 +46,14 @@ import (
 var _ = Describe("replace_misconfigured_pods", func() {
 	var cluster *fdbv1beta2.FoundationDBCluster
 	var log logr.Logger
+	var deprecationOptions internal.DeprecationOptions
 	logf.SetLogger(zap.New(zap.UseDevMode(true), zap.WriteTo(GinkgoWriter)))
 
 	BeforeEach(func() {
 		log = logf.Log.WithName("replacements")
 		cluster = internal.CreateDefaultCluster()
-		Expect(internal.NormalizeClusterSpec(cluster, internal.DeprecationOptions{UseFutureDefaults: true})).NotTo(HaveOccurred())
```

Comment:
did it just not do anything before?

Reply:
Correct, probably a relict when we removed some things during the fdbv1beta1 -> fdbv1beta2 migration.